modules/AWS/nfs/main.tf

```diff
@@ -6,14 +6,13 @@ resource "aws_ebs_volume" "shared_home" {
   type        = local.storage_class
 
   tags = {
-    Name = "${local.product}-nfs-shared-home"
+    Name = "nfs-shared-home"
```

Comment:
Any reason we're removing product from the tag?

Reply:
Right, it should be change to `${var.product}-nfs-shared-home`